ccd-definition/CaseEventToFields/submitApplication.json

```diff
@@ -21,5 +21,17 @@
     "PageID": 1,
     "PageDisplayOrder": 1,
     "PageColumnNumber": 1
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "submitApplication",
+    "CaseFieldID": "sendToCtsc",
+    "PageFieldDisplayOrder": 3,
```

Comment:
This could be potentially be moved up to be PageFieldDisplayOrder 1 to avoid a gap between content and submit buttons.

Reply:
@Jamesnelson117 have you spoken to content about this? I am not sure the declaration of truth should have any other questions on. I would maybe suggest another page?

Reply:
I think it's more a decision for Eilis, i'll ask her.